libs/datamodel/core/tests/reformat/reformat.rs

```diff
@@ -789,14 +789,6 @@ fn multiple_new_lines_between_top_level_elements_must_be_reduced_to_a_single_one
         }
 
 
-        // free floating comment
-        /// free floating doc comment
```

Comment:
I think the free floating comments (this and the next one) should stay

Reply:
Why? We have them before every element we test. These were before the type aliases, but we have plenty of them left in the file.

Reply:
Ah indeed, I missed that and assumed they were there only once. All good then :+1: